bundles/org.openhab.binding.bigassfan/src/main/java/org/openhab/binding/bigassfan/internal/BigAssFanHandlerFactory.java

```diff
@@ -37,6 +39,7 @@
 @Component(service = ThingHandlerFactory.class, configurationPid = "binding.bigassfan")
 public class BigAssFanHandlerFactory extends BaseThingHandlerFactory {
 
+    private final Logger logger = LoggerFactory.getLogger(BigAssFanHandlerFactory.class);
```

Comment:
Why did you add a logger when the logger is not used?

Reply:
Used it for a moment, but removed it and did not remove the decleration. Fixed